draft-ietf-pquip-pqc-hsm-constrained.md

```diff
@@ -94,47 +100,64 @@ informative:
 
 --- abstract
 
-This document offers guidance on incorporating Post-Quantum Cryptography (PQC) into
-resource-constrained devices, including IoT devices and lightweight Hardware Security
-Modules (HSMs), which operate under tight limitations on compute power, memory, storage,
-and energy. It highlights how the Root of Trust acts as the foundation for secure operations,
-enabling features such as seed-based key generation to reduce the need for persistent storage,
-efficient approaches to managing ephemeral keys, and methods for offloading cryptographic tasks
-in low-resource environments. Additionally, it examines how PQC affects firmware update
-mechanisms in such constrained systems.
+This document provides guidance on integrating Post-Quantum Cryptography (PQC) into
+resource-constrained devices, such as IoT nodes and lightweight Hardware Security Modules
+(HSMs). These systems often operate with strict limitations on processing power, RAM, and
```

Comment:
This sentence is repreted in introduction below:

These systems often operate with strict limitations on processing power, RAM, and
flash memory, and may even be battery-powered.